crates/examples/hello.rs

```diff
@@ -0,0 +1,19 @@
+use std::sync::Arc;
+
+use datafusion::{
+    error::Result,
+    prelude::{DataFrame, SessionContext},
+};
+use hudi::HudiDataSource;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let data_path = "/your/hudi/data/path";
```

Comment:
`data_path` variable should ideally not be hardcoded. Instead, you might want to pass it as a command-line argument or environment variable, allowing the example to be more flexible and reusable in different environments. Something like below:
```
data_path = std::env::var("HUDI_DATA_PATH").unwrap_or_else(|_| "/your/hudi/data/path".to_string());
```